controllers/testdata/declarative/case_3/expected/not_created/sample-pipeline.yaml.tmpl

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: sample-pipeline-testdsp3
+    namespace: default
+    labels:
+        app: ds-pipeline-testdsp2
```

```suggestion
        app: ds-pipeline-testdsp3
```
should this be same label as app